packages/core/src/source/LanguageError.ts

```diff
@@ -52,7 +52,7 @@ export namespace LanguageError {
 	}
 }
 
-export const enum ErrorSeverity {
+export enum ErrorSeverity {
```

Comment:
`const enum` is weird. Code don't play nice with it under `"verbatimModuleSyntax": true`.